cmd/server.go

```diff
@@ -218,6 +210,14 @@ Config file values are overridden by environment variables which in turn are ove
 			return s.run()
 		}),
 	}
+
+	// Configure viper to accept env vars prefixed with ATLANTIS_ that can be
+	// used instead of flags.
+	s.Viper.SetEnvPrefix("ATLANTIS")
```

Comment:
@psalaberria002 I moved this in here so that I could test this functionality. I liked it better in `main.go` but the tests don't run through there so this is a compromise.